tests/integration/targets/vmware_ops_vcenter_host_connection_test/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Test
```

Comment:
let's give it more meaningful name, like `Running Host Connection Test`

Reply:
done